gnat2goto/driver/driver.adb

```diff
@@ -226,6 +226,17 @@ package body Driver is
       end;
 
       if not Add_Start then
```

Comment:
Default mode "don't add comments" to this thing I had to figure out what it is doing by looking at the the out parameter in another function. 
Status: "on".  
:)